tests/system/action/motion_comment/test_create.py

```diff
@@ -50,3 +58,35 @@ def test_create_wrong_field(self) -> None:
             "data must not contain {'wrong_field'} properties",
             response.json["message"],
         )
+
+    def test_create_no_permission(self) -> None:
+        self.base_permission_test(
+            self.permission_test_model,
+            "motion_comment.create",
+            {"comment": "test_Xcdfgee", "motion_id": 357, "section_id": 78},
+        )
+
+    def test_create_permission(self) -> None:
+        self.base_permission_test(
+            self.permission_test_model,
+            "motion_comment.create",
+            {"comment": "test_Xcdfgee", "motion_id": 357, "section_id": 78},
+            Permissions.Motion.CAN_SEE,
+        )
+
+    def test_create_no_permission_cause_write_group(self) -> None:
```

Comment:
Please also add a test where the request user is a meeting admin and is not in any of the write request groups, it should suceed.

Reply:
Update the default test to check this, too.